content/src/logic/entity-parser.ts

```diff
@@ -0,0 +1,66 @@
+import { EntityContentItemReference } from '@dcl/hashing'
+import { Entity, EntityType } from '@dcl/schemas'
+import { AnyObject } from '../types'
+
+function getObjectEntityFromBuffer(buffer: Uint8Array): AnyObject {
+  try {
+    return JSON.parse(new TextDecoder().decode(buffer))
```

Comment:
I would instantiate the TextDecoder once at the top of the file and reuse it, so we only do it once